packages/@aws-cdk/aws-stepfunctions-tasks/README.md

```diff
@@ -327,31 +329,60 @@ Step Functions supports [ECS/Fargate](https://docs.aws.amazon.com/step-functions
 
 [RunTask](https://docs.aws.amazon.com/step-functions/latest/dg/connect-ecs.html) starts a new task using the specified task definition.
 
+#### EC2
+
+The following example runs a job from a task definition on EC2
```

Comment:
I barely can see any difference in the code snippets between Fargate and EC2. What's the difference? 

We should provide a brief 2-3 sentence explanation and some links.

Reply:
good callout!

it's not a great example. the primary difference is that fargate is managed and EC2 is not. Users would need to configure placement strategies and constraints for EC2 instances.

Updated the example and a brief lead-in to the differences.